src/flagpole/flagpole_eval.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# flake8: noqa: S002
+
+import argparse
+import dataclasses
+import sys
+from pathlib import Path
+from typing import Any, Tuple, cast
```

Comment:
Looks like Tuple is being imported but you're using `tuple` for typing, not needed